lib/src/repository/check_repository.dart

```diff
@@ -143,9 +143,16 @@ Future<VerifiedRepository?> checkRepository({
               '`$path` from the repository has no `repository` or `homepage` URL.');
         }
         final gitRepoUrl = Repository.tryParseUrl(gitRepoOrHomepage);
-        if (gitRepoUrl?.cloneUrl != parsedSourceUrl.cloneUrl) {
+        if (gitRepoUrl == null) {
+          return _PubspecMatch(
+              path,
+              true,
```

Comment:
Could we make this a named (but required argument) that is almost always nice for boolean parameters